jdk/make/closed/autoconf/custom-hook.m4

```diff
@@ -375,6 +376,25 @@ AC_DEFUN([OPENJ9_CONFIGURE_OPENJDK_METHODHANDLES],
   AC_SUBST(OPENJ9_ENABLE_OPENJDK_METHODHANDLES)
 ])
 
+AC_DEFUN([OPENJ9_CONFIGURE_SNAPSHOTS],
+[
+  AC_MSG_CHECKING([for snapshot support])
+  AC_ARG_ENABLE([snapshots], [AS_HELP_STRING([--enable-snapshots],
+      [enable RAM persistence snapshots @<:@disabled@:>@])])
+  OPENJ9_ENABLE_SNAPSHOTS=false
+  if test "x$enable_snapshots" = xyes ; then
+    AC_MSG_RESULT([yes (explicitly enabled)])
+    OPENJ9_ENABLE_SNAPSHOTS=true
+  elif test "x$enable_snapshots" = xno ; then
+    AC_MSG_RESULT([no (explicit)])
```

Comment:
Why does this not say `"no (explicitly disabled)"` like other options?

Reply:
You're right. This is a mistake. I will open new PR to address any concerns you have here.